xtask/src/ci.rs

```diff
@@ -71,7 +71,10 @@ impl CiTask {
     fn build_nightly(&self) -> xshell::Result<()> {
         let fmt_res = cmd!("rustup run nightly cargo fmt -- --check").run();
         let clippy_res = cmd!("rustup run nightly cargo ruma-clippy -D warnings").run();
-        let sort_res = cmd!("rustup run nightly cargo sort --workspace --grouped --check").run();
+        let sort_res = cmd!(
+            "rustup run nightly cargo sort --workspace --grouped --check --order package,features,dependencies,dev-dependencies,build-dependencies"
```

Comment:
Please make this multiline. `cmd!` accepts multiline strings, no `\` needed, so for example

"
rustup run nightly cargo sort
    --workspace --grouped --check
    --order foo,bar,baz
"

Reply:
I knew there was a way to do it I only tried `\`and it didn't work, thanks.